scipy/optimize/tests/test_linprog.py

```diff
@@ -11,6 +11,11 @@
 from scipy._lib._numpy_compat import _assert_warns, suppress_warnings
 from scipy.sparse.linalg import MatrixRankWarning
 from scipy.linalg import LinAlgWarning
+try:
```

Comment:
Will this test both umfpack and cholmod?

Reply:
It tests UMFPACK in the builds that install the scikit. I don't think there are any now with scikit-sparse. Of course, if there is an issue using CHOLMOD on some machine then the code will just move on to UMFPACK.